Testing/__tests__/Contract/ContractInstance.spec.ts

```diff
@@ -0,0 +1,78 @@
+import BN from 'bn.js';
+import { ISimpleContract, ISimpleContractConnected, IComplexContract, IComplexContractConnected } from '../../contracts/abiTypes';
+import { ProxiedNode, CreateContract, ContractInstance } from '../../../src/lib';
```

Comment:
Baseurl is set the src in the tsconfig, you should be able to do `...from lib/`, you might need to include `__tests__` as an included path in the tsconfig though